lib/internal/validators.js

```diff
@@ -404,7 +404,7 @@ function validateUnion(value, name, union) {
 }
 
 function validateLinkHeaderValue(value, name) {
-  const linkValueRegExp = /^(?:<[^>]*>;)\s*(?:rel=(")?[^;"]*\1;?)\s*(?:(?:as|anchor|title)=(")?[^;"]*\2)?$/;
+  const linkValueRegExp = /^(?:<[^>]*>;)\s*(?:rel=(")?[^;"]*\1;?)\s*(?:(?:as|anchor|title|crossorigin|disabled|fetchpriority|rel)=(")?[^;"]*\2)?$/;
```

Comment:
shouldn't we add `referrerpolicy` too?

Reply:
Thanks for the review. You're right. I just committed the change.